src/validations/review.validation.ts

```diff
@@ -0,0 +1,29 @@
+import Joi from 'joi';;
+
+export const reviewVal = {
+
+  createReview: Joi.object({
+    title: Joi.string().min(1).required().messages({
+      'string.min': 'Title must contain at least 1 character',
+    }),
+    comment: Joi.string().min(1).required().messages({
+      'string.min': 'Comment must contain at least 1 character'
+    }),
+    rating: Joi.number().integer().min(1).max(5).required().messages({
+      'number.base': 'Rating must be a number',
+      'number.integer': 'Rating must be an integer',
+      'any.required': 'Rating is a required field',
+    }),
+  }),
+  updateReview: Joi.object({
+    title: Joi.string().min(1).optional().messages({
+      'string.empty': 'Title cannot be empty',
+      'string.min': 'Title must contain at least 1 character',
+    }),
+    comment: Joi.string().min(1).optional().messages({
+      'string.empty': 'Comment cannot be empty',
+      'string.min': 'Comment must contain at least 1 character',
+    }),
+    rating: Joi.number().integer().min(1).max(5).optional()
+  }),
+};
```

Comment:
You don't have to provide all messages like this. In most cases the ones provided by joi by default are understandable